modules/openapi-generator/src/main/resources/rust/reqwest/configuration.mustache

```diff
@@ -1,16 +1,18 @@
 {{>partial_header}}
 
+use serde::Deserialize;
 {{#withAWSV4Signature}}
 use std::time::SystemTime;
 use aws_sigv4::http_request::{sign, SigningSettings, SigningParams, SignableRequest};
 use http;
 use secrecy::{SecretString, ExposeSecret};
 {{/withAWSV4Signature}}
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Deserialize)]
```

Comment:
This allows users to read configuration from a file, which I imagine is a common use case.